UPGRADE.md

```diff
@@ -1,13 +1,7 @@
-## Upgrade steps from v. 5.2.x to v. 5.3 for native/hybrid/react native apps
+## Upgrade steps from v. 5.3.x to v. 6.0 for native/hybrid/react native apps
 
 To upgrade native, hybrid and react native apps, we strongly recommend creating a new app from the app templates in [the forceios npm package](https://npmjs.org/package/forceios), then migrating the artifacts specific to your app into the new template. The other recommended approach would be to upgrade using [Cocoapods](https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Specs).
```

Comment:
Fro hybrid apps, they will need to use forcehybrid. And for react native apps, they should use forcereact.